wordpress-mobile/WordPress-iOS
# [Stats Refresh] Post Stats: add Recent Weeks

Ref #11189

This adds the Recent Weeks card to the Post Stats view. It shows total views for the last 6 weeks. Expanding a week shows the total views for each day in the week.

To test:

- Access Post Stats via:
  - Insights > Latest Post Summary > View more
  - Period > Posts and Pages > row selection
  - Period > Posts and Pages > View more > row selection
- Verify Recent Weeks appears, and each week expands to show week days.

<img width="350" alt="recent_weeks" src="https://user-images.githubusercontent.com/1816888/56518778-8ad18080-64fd-11e9-886b-bd4626649269.png">

<img width="350" alt="recent_weeks_expanded" src="https://user-images.githubusercontent.com/1816888/56518781-8dcc7100-64fd-11e9-996d-1041ccab36f4.png">
